backend/core/domains/contracts/views.py

```
@@ -375,8 +388,72 @@ def for_event(self, request):
+                EventContractDetailSerializer(updated_contract).data,
+                status=status.HTTP_200_OK
+            )
+            
+        except Exception as e:
+            return Response(
+                {'error': f'Failed to send contract: {str(e)}'},
+                status=status.HTTP_500_INTERNAL_SERVER_ERROR
+            )
+    
+    @action(detail=True, methods=['get'])
+    def download_pdf(self, request, pk=None):
```

Ruff (0.13.3)

Unused method argument: `request`

(ARG002)